### PR TITLE
Add ops-iam 20220929 minutes

### PR DESCRIPTION
**Pull request by joshmue**
Signed-off-by: Joshua Mühlfort <muehlfort@gonicus.de>

### CODE REVIEW

File: ops-iam/20220929.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### Moving from R3 to R4 cycle
+https://github.com/SovereignCloudStack/issues/issues?q=is%3Aopen+is%3Aissue+label%3AOps+milestone%3A%22R3+%28v4.0.0%29%22
+=> OVN Issues marked as R3, discussion on wether to move them to R4 or R5.
+
+@garloff raises the point, that in the past, this was something vital for the Operators so that if they need it, we should see that we deliver this as R4 and not just move it to R5.
+We should talk to PCO, since they are heavy OVN user.
+
+### machine_identity (@joshmue)
+https://github.com/SovereignCloudStack/Docs/pull/140
```
Comment (garloff) [inline ops-iam/20220929.md:28]: Add-in: We agreed that we want some kind of ADR process and establish and use it during the R4 cycle. We had a discussion on how ADRs would be reviewed and decided upon. First ADR should actually document what the (lazy) consensus and decision process is. @horazont volunteerd to make a proposal.

**Review by garloff — CHANGES_REQUESTED** [review]

File: ops-iam/20220929.md (hunk 1 above)
Comment (fkr) [reply ops-iam/20220929.md:28]: thanks for the addition @garloff

Comment (horazont) [reply ops-iam/20220929.md:28]: https://github.com/SovereignCloudStack/Docs/pull/143

Comment (joshmue) [reply ops-iam/20220929.md:28]: Incorperated that into the minutes.

**Review by fkr — APPROVED** [review]
LGTM! thanks @joshmue!

**Comment by fkr** [comment]
I'll merge this. @joshmue did the requested changes and @garloff is on PTA.

**Pull request merged**